Cargo.toml

```diff
@@ -70,8 +70,8 @@ neard = { path = "./neard"}
 
 testlib = { path = "./test-utils/testlib" }
 
-[replace]
-"ethereum-types:0.10.0" = { path = "./patches/ethereum-types-0.10.0-to-0.11.0" }
+[patch.crates-io]
+ethereum-types = { path = "./patches/ethereum-types-0.10.0-to-0.11.0" }
```

Comment:
It is weird if this works fine since it does not specify that it needs to only patch the 0.10.0 version :thinking:

Reply:
Not a lot changes, I think. Plus, `[replace]` is [deprecated](https://doc.rust-lang.org/cargo/reference/overriding-dependencies.html#the-replace-section).

```patch
diff --git a/Cargo.toml b/Cargo.toml
index 8fd5c4e1..d6bbb532 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -70,8 +70,8 @@ neard = { path = "./neard"}
 
 testlib = { path = "./test-utils/testlib" }
 
-[replace]
-"ethereum-types:0.10.0" = { path = "./patches/ethereum-types-0.10.0-to-0.11.0" }
+[patch.crates-io]
+ethereum-types = { path = "./patches/ethereum-types-0.10.0-to-0.11.0" }
 
 [profile.release]
 lto = true        # Enable full link-time optimization.
diff --git a/Cargo.lock b/Cargo.lock
index 17dbadda..bd315595 100644
--- a/Cargo.lock
+++ b/Cargo.lock
@@ -1694,7 +1694,7 @@ source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "53d4e679d6864bc26210feb5cf044e245741cd9d7701b35c00440a6e84d61399"
 dependencies = [
  "anyhow",
- "ethereum-types 0.10.0 (registry+https://github.com/rust-lang/crates.io-index)",
+ "ethereum-types 0.10.0",
  "hex",
  "serde",
  "serde_json",
@@ -1743,13 +1743,6 @@ dependencies = [
  "ethereum-types 0.11.0",
 ]
 
-[[package]]
-name = "ethereum-types"
-version = "0.10.0"
-source = "registry+https://github.com/rust-lang/crates.io-index"
-checksum = "05dc5f0df4915fa6dff7f975a8366ecfaaa8959c74235469495153e7bb1b280e"
-replace = "ethereum-types 0.10.0"
-
 [[package]]
 name = "ethereum-types"
 version = "0.11.0"
```